R/expect-datacomp.R

```diff
@@ -124,6 +125,7 @@ expect_subset <- function(data2,
                           flt = TRUE,
                           data = get_testdata()) {
 
+  check_expect_data_pipe(enquo(vars))
```

Comment:
There's no `vars` argument to `expect_subset`

Reply:
Weird that didn't throw an error in the tests (maybe because of `enquo()`?). Fixed!